basket/news/tasks.py

```diff
@@ -879,7 +880,7 @@ def process_subhub_event_subscription_charge(data):
             # if the following succeeds, we don't have to worry about setting
             # the initial purchase flag - we leave it alone, assuming it was
             # set properly in the except block below
-            sfdc.opportunity.update('PMT_Transaction_ID__c/{}'.format(charge_id), transaction_data)
+            sfdc.opportunity.update(f'PMT_Transaction_ID__c/{charge_id}', transaction_data)
```

Comment:
noting that this will prob throw a merge conflict now that the API has been majorly refactored. (we are no longer doing any updates, so either way can revert this.)

Reply:
yeah. I got a bit carried away with f-strings. I'll revert this. But yay f-strings 🎉